tools/analysis/analysis_test.go

```diff
@@ -256,3 +256,42 @@ func TestCheckError(t *testing.T) {
 	var checkerError *sema.CheckerError
 	require.ErrorAs(t, err, &checkerError)
 }
+
+func TestStdlib(t *testing.T) {
+
+	t.Parallel()
+
+	location := common.ScriptLocation{}
+
+	const code = `
+	  pub fun main() {
+          panic("test")
+      }
+	`
+
+	config := &analysis.Config{
+		Mode: analysis.NeedTypes,
+		ResolveCode: func(
+			location common.Location,
+			importingLocation common.Location,
+			importRange ast.Range,
+		) ([]byte, error) {
+			switch location {
+			case location:
+				return []byte(code), nil
+
+			default:
+				require.FailNow(t,
+					"import of unknown location: %s",
+					"location: %s",
+					location,
+				)
+				return nil, nil
```

Comment:
this seems unreachable, did you mean to shadow the `location` bound at line 264?

Reply:
Good catch! Renamed the outer variable and didn't realize it got shadowed

Reply:
Fixes in aeb2ee1649add07f5b73905ea61468723ca99f0f